# Russian translation for duplicity
# Copyright (c) 2010 Rosetta Contributors and Canonical Ltd 2010
# This file is distributed under the same license as the duplicity package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2010.
#
msgid ""
msgstr ""
"Project-Id-Version: duplicity\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2016-02-18 16:51+0000\n"
"PO-Revision-Date: 2010-03-11 14:36+0000\n"
"Last-Translator: Kenneth Loafman <kenneth@loafman.com>\n"
"Language-Team: Russian <ru@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=n%10==1 && n%100!=11 ? 0 : n%10>=2 && "
"n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2;\n"
"X-Launchpad-Export-Date: 2016-02-19 05:11+0000\n"
"X-Generator: Launchpad (build 17925)\n"

#: ../bin/duplicity:130
msgid "Reuse configured PASSPHRASE as SIGN_PASSPHRASE"
msgstr "Повторное использование настроенного PASSPHRASE как SIGN_PASSPHRASE"

#: ../bin/duplicity:137
msgid "Reuse configured SIGN_PASSPHRASE as PASSPHRASE"
msgstr "Повторное использование настроенного SIGN_PASSPHRASE как PASSPHRASE"

#: ../bin/duplicity:178
msgid "PASSPHRASE variable not set, asking user."
msgstr "Переменная PASSPHRASE не назначена, выполняется запрос пользователя."

#: ../bin/duplicity:193
msgid "GnuPG passphrase for signing key:"
msgstr "Кодовая фраза GnuPG для подписывающего ключа:"

#: ../bin/duplicity:198
msgid "GnuPG passphrase:"
msgstr "Кодовая фраза GnuPG:"

#: ../bin/duplicity:203
msgid "Retype passphrase for signing key to confirm: "
msgstr "Введите повторно кодовую фразу для подписывающего ключа: "

#: ../bin/duplicity:205
msgid "Retype passphrase to confirm: "
msgstr "Введите кодовую фразу для подтверждения снова: "

#: ../bin/duplicity:208
msgid "First and second passphrases do not match!  Please try again."
msgstr ""
"Первая и вторая кодовая фраза не совпадают! Пожалуйста, повторите снова."

#: ../bin/duplicity:213
msgid ""
"Cannot use empty passphrase with symmetric encryption!  Please try again."
msgstr ""
"Невозможно использовать пустую кодовую фразу с симметричным шифрованием!  "
"Пожалуйста, повторите снова."

#: ../bin/duplicity:269
#, python-format
msgid ""
"File %s complete in backup set.\n"
"Continuing restart on file %s."
msgstr ""
"Файл %s уже имеется в наборе резервного копирования.\n"
"Продолжение перезапуска на файле %s."

#: ../bin/duplicity:278
#, python-format
msgid ""
"File %s missing in backup set.\n"
"Continuing restart on file %s."
msgstr ""
"Файл %s отсутствует в наборе резервного копирования.\n"
"Продолжение перезапуска на файле %s."

#: ../bin/duplicity:327
#, python-format
msgid "File %s was corrupted during upload."
msgstr "Файл %s был повреждён во время выгрузки."

#: ../bin/duplicity:360
msgid ""
"Restarting backup, but current encryption settings do not match original "
"settings"
msgstr ""
"Перезапуск резервного копирования, но настройки текущего шифрования не "
"соответствуют исходным настройкам"

#: ../bin/duplicity:383
#, python-format
msgid "Restarting after volume %s, file %s, block %s"
msgstr ""

#: ../bin/duplicity:451
#, python-format
msgid "Processed volume %d"
msgstr ""

#: ../bin/duplicity:601
msgid ""
"Fatal Error: Unable to start incremental backup.  Old signatures not found "
"and incremental specified"
msgstr ""
"Фатальная ошибка: Не удалось начать пошаговое резервное копирование. Старая "
"сигнатура не найдена и указан пошаговый режим"

#: ../bin/duplicity:605
msgid "No signatures found, switching to full backup."
msgstr "Сигнатуры не найдены, переключение на полную резервную копию."

#: ../bin/duplicity:619
msgid "Backup Statistics"
msgstr "Статистика резервного копирования"

#: ../bin/duplicity:695
#, python-format
msgid "%s not found in archive, no files restored."
msgstr "%s не найден в архиве, файлы не восстановлены."

#: ../bin/duplicity:728
msgid "No files found in archive - nothing restored."
msgstr "Ничего не найдено в архиве - ничего не восстановлено."

#: ../bin/duplicity:761
#, python-format
msgid "Processed volume %d of %d"
msgstr "Обработан том %d из %d"

#: ../bin/duplicity:795
#, python-format
msgid "Invalid data - %s hash mismatch for file:"
msgstr "Недопустимые данные - %s hash не соответствует файлу:"

#: ../bin/duplicity:797
#, python-format
msgid "Calculated hash: %s"
msgstr "Вычисленный хэш: %s"

#: ../bin/duplicity:798
#, python-format
msgid "Manifest hash: %s"
msgstr "Хеш манифеста: %s"

#: ../bin/duplicity:837
#, python-format
msgid "Volume was signed by key %s, not %s"
msgstr "Том был подписан ключом %s, а не %s"

#: ../bin/duplicity:869
#, python-format
msgid "Verify complete: %s, %s."
msgstr "Проверка завершена: %s, %s."

#: ../bin/duplicity:870
#, python-format
msgid "%d file compared"
msgid_plural "%d files compared"
msgstr[0] "%d файл сравнён"
msgstr[1] "%d файла сравнены"
msgstr[2] "%d файлов сравнено"

#: ../bin/duplicity:872
#, python-format
msgid "%d difference found"
msgid_plural "%d differences found"
msgstr[0] "%d отличие найдено"
msgstr[1] "%d отличия найдено"
msgstr[2] "%d отличий найдено"

#: ../bin/duplicity:891
msgid "No extraneous files found, nothing deleted in cleanup."
msgstr "Посторонних файлов не найдено, при очистке ничего не удалено."

#: ../bin/duplicity:896
msgid "Deleting this file from backend:"
msgid_plural "Deleting these files from backend:"
msgstr[0] "Удаляется следующий файл из заглушки:"
msgstr[1] "Удаляются следующие файлы из заглушки:"
msgstr[2] "Удаляются следующие файлы из заглушки:"

#: ../bin/duplicity:908
msgid "Found the following file to delete:"
msgid_plural "Found the following files to delete:"
msgstr[0] "Найден файл для удаления:"
msgstr[1] "Найдены файлы для удаления:"
msgstr[2] "Найдены файлы для удаления:"

#: ../bin/duplicity:912
msgid "Run duplicity again with the --force option to actually delete."
msgstr ""
"Запустите duplicity ещё раз с ключом --force чтобы удалить на самом деле."

#: ../bin/duplicity:955
msgid "There are backup set(s) at time(s):"
msgstr "Есть резервные копии за следующие даты:"

#: ../bin/duplicity:957
msgid "Which can't be deleted because newer sets depend on them."
msgstr "Какие не могут быть удалены из-за новых наборов зависящих от них."

#: ../bin/duplicity:961
msgid ""
"Current active backup chain is older than specified time.  However, it will "
"not be deleted.  To remove all your backups, manually purge the repository."
msgstr ""
"Текущая активная цепочка резервных копий старее чем указанное время.  Те не "
"менее, она не будет удалена. Чтобы удалить все резервные копии, вручную "
"очистите хранилище."

#: ../bin/duplicity:967
msgid "No old backup sets found, nothing deleted."
msgstr "Старые резервные копии не найдены, ничего не удалено."

#: ../bin/duplicity:970
msgid "Deleting backup chain at time:"
msgid_plural "Deleting backup chains at times:"
msgstr[0] "Удаление цепочки резервных копий по времени:"
msgstr[1] "Удаление цепочек резервных копий по времени:"
msgstr[2] "Удаление цепочек резервных копий по времени:"

#: ../bin/duplicity:981
#, python-format
msgid "Deleting incremental signature chain %s"
msgstr ""

#: ../bin/duplicity:983
#, python-format
msgid "Deleting incremental backup chain %s"
msgstr ""

#: ../bin/duplicity:986
#, python-format
msgid "Deleting complete signature chain %s"
msgstr ""

#: ../bin/duplicity:988
#, python-format
msgid "Deleting complete backup chain %s"
msgstr "Удаление полной цепочки резервных копий  %s"

#: ../bin/duplicity:994
msgid "Found old backup chain at the following time:"
msgid_plural "Found old backup chains at the following times:"
msgstr[0] ""
msgstr[1] ""

#: ../bin/duplicity:998
msgid "Rerun command with --force option to actually delete."
msgstr "Rerun command with --force option to actually delete."

#: ../bin/duplicity:1075
#, python-format
msgid "Deleting local %s (not authoritative at backend)."
msgstr ""
"Удаление локального %s (недостаточно надёжный во внутреннем интерфейсе)."

#: ../bin/duplicity:1079
#, python-format
msgid "Unable to delete %s: %s"
msgstr "Невозможно удалить %s: %s"

#: ../bin/duplicity:1109 ../duplicity/dup_temp.py:265
#, python-format
msgid "Failed to read %s: %s"
msgstr "Не удалось прочитать %s: %s"

#: ../bin/duplicity:1123
#, python-format
msgid "Copying %s to local cache."
msgstr "Копируется %s в локальный кэш."

#: ../bin/duplicity:1171
msgid "Local and Remote metadata are synchronized, no sync needed."
msgstr ""
"Локальные и удалённые метаданные синхронизированы, синхронизация не "
"требуется."

#: ../bin/duplicity:1176
msgid "Synchronizing remote metadata to local cache..."
msgstr ""
"Синхронизация удалённых метаданных с локальными временными файлами..."

#: ../bin/duplicity:1191
msgid "Sync would copy the following from remote to local:"
msgstr "Синхронизация скопирует это из удаленного места в локальное:"

#: ../bin/duplicity:1194
msgid "Sync would remove the following spurious local files:"
msgstr "Синхронизация удалит следующие  неизвестные файлы:"

#: ../bin/duplicity:1237
msgid "Unable to get free space on temp."
msgstr ""
"Не удалось узнать объем доступного пространства во временном каталоге."

#: ../bin/duplicity:1245
#, python-format
msgid "Temp space has %d available, backup needs approx %d."
msgstr ""
"Во временно каталоге доступного %d, для резервного копирования нужно "
"примерно %d."

#: ../bin/duplicity:1248
#, python-format
msgid "Temp has %d available, backup will use approx %d."
msgstr ""
"Во временном каталоге доступно %d, для резервного копирования будет "
"использовано примерно %d."

#: ../bin/duplicity:1256
msgid "Unable to get max open files."
msgstr "Не удалось узнать максимальное количество открытых файлов."

#: ../bin/duplicity:1260
#, python-format
msgid ""
"Max open files of %s is too low, should be >= 1024.\n"
"Use 'ulimit -n 1024' or higher to correct.\n"
msgstr ""
"Максимальное количество открытых файлов %s слишком мало, Должно быть >= "
"1024.\n"
"Используйте 'ulimit -n 1024' или выше, чтобы исправить.\n"

#: ../bin/duplicity:1311
msgid ""
"RESTART: The first volume failed to upload before termination.\n"
"         Restart is impossible...starting backup from beginning."
msgstr ""
"ПЕРЕЗАПУСК: Ошибка выгрузки первого тома перед завершением работы.\n"
"         Перезапуск невозможен...выполнение резервного копирования с начала."

#: ../bin/duplicity:1317
#, python-format
msgid ""
"RESTART: Volumes %d to %d failed to upload before termination.\n"
"         Restarting backup at volume %d."
msgstr ""
"ПЕРЕЗАПУСК: Ошибка выгрузки томов %d дo %d перед завершением работы.\n"
"         Перезапуск резервного копирования в томе %d."

#: ../bin/duplicity:1324
#, python-format
msgid ""
"RESTART: Impossible backup state: manifest has %d vols, remote has %d vols.\n"
"         Restart is impossible ... duplicity will clean off the last "
"partial\n"
"         backup then restart the backup from the beginning."
msgstr ""
"ПЕРЕЗАПУСК: Невозможное состояние резервной копии: манифест содержит %d "
"томов, удалённый содержит %d томов.\n"
"         Перезапуск невозможен ... duplicity произведёт очистку последней "
"части\n"
"         резервной копии, затем запустит резервное копирования сначала."

#: ../bin/duplicity:1345
msgid ""
"\n"
"PYTHONOPTIMIZE in the environment causes duplicity to fail to\n"
"recognize its own backups.  Please remove PYTHONOPTIMIZE from\n"
"the environment and rerun the backup.\n"
"\n"
"See https://bugs.launchpad.net/duplicity/+bug/931175\n"
msgstr ""

#: ../bin/duplicity:1330
msgid ""
"\n"
"Duplicity 0.6 series is being deprecated:\n"
"See http://www.nongnu.org/duplicity/\n"
msgstr ""

#: ../bin/duplicity:1428
#, python-format
msgid "Last %s backup left a partial set, restarting."
msgstr ""
"Последняя %s резервная копия оставила частичный набор, выполняется "
"перезапуск."

#: ../bin/duplicity:1432
#, python-format
msgid "Cleaning up previous partial %s backup set, restarting."
msgstr ""
"Очистка предыдущего частичного %s набора резервного копирования, выполняется "
"перезапуск."

#: ../bin/duplicity:1443
msgid "Last full backup date:"
msgstr "Время последней полной резервной копии:"

#: ../bin/duplicity:1445
msgid "Last full backup date: none"
msgstr "Время последней полной резервной копии: нету"

#: ../bin/duplicity:1447
msgid "Last full backup is too old, forcing full backup"
msgstr ""
"Последняя полная резервная копия слишком старая, будет сделана полная "
"резервная копия"

#: ../bin/duplicity:1493
msgid ""
"When using symmetric encryption, the signing passphrase must equal the "
"encryption passphrase."
msgstr ""

#: ../bin/duplicity:1546
msgid "INT intercepted...exiting."
msgstr "Перехват INT...выполняется выход."

#: ../bin/duplicity:1554
#, python-format
msgid "GPG error detail: %s"
msgstr "Подробности ошибки GPG: %s"

#: ../bin/duplicity:1564
#, python-format
msgid "User error detail: %s"
msgstr "Сведения ошибки пользователя: %s"

#: ../bin/duplicity:1574
#, python-format
msgid "Backend error detail: %s"
msgstr "Подробности об ошибке в заглушке: %s"

#: ../bin/rdiffdir:61 ../duplicity/commandline.py:259
#, python-format
msgid "Error opening file %s"
msgstr "Ошибка открытия файла %s"

#: ../bin/rdiffdir:128
#, python-format
msgid "File %s already exists, will not overwrite."
msgstr ""

#: ../duplicity/selection.py:127
#, python-format
msgid "Skipping socket %s"
msgstr "Пропуск сокета %s"

#: ../duplicity/selection.py:131
#, python-format
msgid "Error initializing file %s"
msgstr "Ошибка инициализации файла %s"

#: ../duplicity/selection.py:135 ../duplicity/selection.py:157
#, python-format
msgid "Error accessing possibly locked file %s"
msgstr "Ошибка доступа к возможно заблокированному файлу %s"

#: ../duplicity/selection.py:172
#, python-format
msgid "Warning: base %s doesn't exist, continuing"
msgstr "Внимание: база %s не существует, выполняется продолжение"

#: ../duplicity/selection.py:175 ../duplicity/selection.py:193
#: ../duplicity/selection.py:196
#, python-format
msgid "Selecting %s"
msgstr "Выбор %s"

#: ../duplicity/selection.py:293
#, python-format
msgid ""
"Fatal Error: The file specification\n"
"    %s\n"
"cannot match any files in the base directory\n"
"    %s\n"
"Useful file specifications begin with the base directory or some\n"
"pattern (such as '**') which matches the base directory."
msgstr ""
"Критическая ошибка: Спецификация файла\n"
"    %s\n"
"не может соответствовать любым файлам в базовом каталоге\n"
"    %s\n"
"Полезные спецификации файла начинаются с базового каталога или\n"
"маски (например '**'), которая соответствует базовому каталогу."

#: ../duplicity/selection.py:302
#, python-format
msgid ""
"Fatal Error while processing expression\n"
"%s"
msgstr ""
"Критическая ошибка при обработке выражения\n"
"%s"

#: ../duplicity/selection.py:312
#, python-format
msgid ""
"Last selection expression:\n"
"    %s\n"
"only specifies that files be included.  Because the default is to\n"
"include all files, the expression is redundant.  Exiting because this\n"
"probably isn't what you meant."
msgstr ""
"Выражение последнего выделения:\n"
"    %s\n"
"только указывает на то, что файлы будут включены.  Так как\n"
"по умолчанию включаются все файлы, следовательно выражение избыточно.  \n"
"Выполнение выхода, так как возможно, что вероятно, это не то, что вы имели в "
"виду."

#: ../duplicity/selection.py:311
#, python-format
msgid "Reading filelist %s"
msgstr "Чтение списка файлов %s"

#: ../duplicity/selection.py:314
#, python-format
msgid "Sorting filelist %s"
msgstr "Сортировка списка файлов %s"

#: ../duplicity/selection.py:341
#, python-format
msgid ""
"Warning: file specification '%s' in filelist %s\n"
"doesn't start with correct prefix %s.  Ignoring."
msgstr ""
"Внимание: спецификация файла '%s' в списке файлов %s\n"
"не начинается с правильного префикса %s.  Пропускание."

#: ../duplicity/selection.py:345
msgid "Future prefix errors will not be logged."
msgstr "Последующие ошибки префикса не будут записываться в журнал событий."

#: ../duplicity/selection.py:361
#, python-format
msgid "Error closing filelist %s"
msgstr "Ошибка закрытия списка файлов %s"

#: ../duplicity/selection.py:368
#, python-format
msgid "Reading globbing filelist %s"
msgstr "Чтение подстановочного списка файлов %s"

#: ../duplicity/selection.py:401
#, python-format
msgid "Error compiling regular expression %s"
msgstr "Ошибка компилирования регулярного выражения %s"

#: ../duplicity/selection.py:418
msgid ""
"Warning: exclude-device-files is not the first selector.\n"
"This may not be what you intended"
msgstr ""
"Внимание: exclude-device-files не является первым выборщиком.\n"
"возможно, это не то, что вы хотели"

#: ../duplicity/commandline.py:71
#, python-format
msgid ""
"Warning: Option %s is pending deprecation and will be removed in a future "
"release.\n"
"Use of default filenames is strongly suggested."
msgstr ""
"Внимание: возможность %s почти устарела и будет выведена из эксплуатации в "
"следующем выпуске.\n"
"Настоятельно предлагаем использовать предложенные имена файлов."

#: ../duplicity/commandline.py:216
#, python-format
msgid "Unable to load gio backend: %s"
msgstr ""

#. Used in usage help to represent a Unix-style path name. Example:
#. --archive-dir <path>
#: ../duplicity/commandline.py:280 ../duplicity/commandline.py:290
#: ../duplicity/commandline.py:307 ../duplicity/commandline.py:381
#: ../duplicity/commandline.py:399 ../duplicity/commandline.py:591
#: ../duplicity/commandline.py:624 ../duplicity/commandline.py:810
msgid "path"
msgstr "путь"

#. Used in usage help to represent an ID for a GnuPG key. Example:
#. --encrypt-key <gpg_key_id>
#. Used in usage help to represent an ID for a hidden GnuPG key. Example:
#. --hidden-encrypt-key <gpg_key_id>
#. Used in usage help to represent an ID for a GnuPG key. Example:
#. --encrypt-key <gpg_key_id>
#: ../duplicity/commandline.py:302 ../duplicity/commandline.py:309
#: ../duplicity/commandline.py:405 ../duplicity/commandline.py:575
#: ../duplicity/commandline.py:783
msgid "gpg-key-id"
msgstr "gpg-key-id"

#. Used in usage help to represent a "glob" style pattern for
#. matching one or more files, as described in the documentation.
#. Example:
#. --exclude <shell_pattern>
#: ../duplicity/commandline.py:317 ../duplicity/commandline.py:430
#: ../duplicity/commandline.py:833
msgid "shell_pattern"
msgstr "shell_pattern"

#. Used in usage help to represent the name of a file. Example:
#. --log-file <filename>
#: ../duplicity/commandline.py:323 ../duplicity/commandline.py:332
#: ../duplicity/commandline.py:339 ../duplicity/commandline.py:432
#: ../duplicity/commandline.py:439 ../duplicity/commandline.py:452
#: ../duplicity/commandline.py:779
msgid "filename"
msgstr "имя файла"

#. Used in usage help to represent a regular expression (regexp).
#: ../duplicity/commandline.py:346 ../duplicity/commandline.py:443
msgid "regular_expression"
msgstr "regular_expression"

#. Used in usage help to represent a time spec for a previous
#. point in time, as described in the documentation. Example:
#. duplicity remove-older-than time [options] target_url
#: ../duplicity/commandline.py:350 ../duplicity/commandline.py:393
#: ../duplicity/commandline.py:514 ../duplicity/commandline.py:865
msgid "time"
msgstr "время"

#. Used in usage help. (Should be consistent with the "Options:"
#. header.) Example:
#. duplicity [full|incremental] [options] source_dir target_url
#: ../duplicity/commandline.py:401 ../duplicity/commandline.py:494
#: ../duplicity/commandline.py:517 ../duplicity/commandline.py:583
#: ../duplicity/commandline.py:798
msgid "options"
msgstr "параметры"

#: ../duplicity/commandline.py:416
#, python-format
msgid ""
"Running in 'ignore errors' mode due to %s; please re-consider if this was "
"not intended"
msgstr ""
"Выполнение в режиме 'пропуска ошибок' в связи с %s; пожалуйста, пересмотрите "
"решение, если это не было намерением"

#. Used in usage help to represent an imap mailbox
#: ../duplicity/commandline.py:428
msgid "imap_mailbox"
msgstr "imap_mailbox"

#: ../duplicity/commandline.py:446
msgid "file_descriptor"
msgstr "file_descriptor"

#. Used in usage help to represent a desired number of
#. something. Example:
#. --num-retries <number>
#: ../duplicity/commandline.py:457 ../duplicity/commandline.py:479
#: ../duplicity/commandline.py:491 ../duplicity/commandline.py:500
#: ../duplicity/commandline.py:541 ../duplicity/commandline.py:546
#: ../duplicity/commandline.py:550 ../duplicity/commandline.py:619
#: ../duplicity/commandline.py:793
msgid "number"
msgstr "количество"

#. Used in usage help (noun)
#: ../duplicity/commandline.py:460
msgid "backup name"
msgstr "название резервной копии"

#. noun
#: ../duplicity/commandline.py:559 ../duplicity/commandline.py:562
#: ../duplicity/commandline.py:764
msgid "command"
msgstr "команда"

#: ../duplicity/commandline.py:565
msgid "pyrax|cloudfiles"
msgstr ""

#: ../duplicity/commandline.py:546
msgid "paramiko|pexpect"
msgstr "paramiko|pexpect"

#: ../duplicity/commandline.py:586
msgid "pem formatted bundle of certificate authorities"
msgstr ""

#. Used in usage help. Example:
#. --timeout <seconds>
#: ../duplicity/commandline.py:596 ../duplicity/commandline.py:827
msgid "seconds"
msgstr "секунды"

#. abbreviation for "character" (noun)
#: ../duplicity/commandline.py:602 ../duplicity/commandline.py:761
msgid "char"
msgstr "символ"

#: ../duplicity/commandline.py:727
#, python-format
msgid "Using archive dir: %s"
msgstr "Использование каталога архива: %s"

#: ../duplicity/commandline.py:728
#, python-format
msgid "Using backup name: %s"
msgstr "Использование наименования резервной копии: %s"

#: ../duplicity/commandline.py:735
#, python-format
msgid "Command line error: %s"
msgstr "Ошибка в командной строке: %s"

#: ../duplicity/commandline.py:736
msgid "Enter 'duplicity --help' for help screen."
msgstr "Введите  'duplicity --help' для экрана справки."

#. Used in usage help to represent a Unix-style path name. Example:
#. rsync://user[:password]@other_host[:port]//absolute_path
#: ../duplicity/commandline.py:749
msgid "absolute_path"
msgstr "полный_путь"

#. Used in usage help. Example:
#. tahoe://alias/some_dir
#: ../duplicity/commandline.py:753
msgid "alias"
msgstr "псевдоним"

#. Used in help to represent a "bucket name" for Amazon Web
#. Services' Simple Storage Service (S3). Example:
#. s3://other.host/bucket_name[/prefix]
#: ../duplicity/commandline.py:758
msgid "bucket_name"
msgstr "bucket_name"

#. Used in usage help to represent the name of a container in
#. Amazon Web Services' Cloudfront. Example:
#. cf+http://container_name
#: ../duplicity/commandline.py:769
msgid "container_name"
msgstr "container_name"

#. noun
#: ../duplicity/commandline.py:772
msgid "count"
msgstr "количество"

#. Used in usage help to represent the name of a file directory
#: ../duplicity/commandline.py:775
msgid "directory"
msgstr "каталог"

#. Used in usage help, e.g. to represent the name of a code
#. module. Example:
#. rsync://user[:password]@other.host[:port]::/module/some_dir
#: ../duplicity/commandline.py:788
msgid "module"
msgstr "модуль"

#. Used in usage help to represent an internet hostname. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/commandline.py:802
msgid "other.host"
msgstr "other.host"

#. Used in usage help. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/commandline.py:806
msgid "password"
msgstr "пароль"

#. Used in usage help to represent a TCP port number. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/commandline.py:814
msgid "port"
msgstr "порт"

#. Used in usage help. This represents a string to be used as a
#. prefix to names for backup files created by Duplicity. Example:
#. s3://other.host/bucket_name[/prefix]
#: ../duplicity/commandline.py:819
msgid "prefix"
msgstr "префикс"

#. Used in usage help to represent a Unix-style path name. Example:
#. rsync://user[:password]@other.host[:port]/relative_path
#: ../duplicity/commandline.py:823
msgid "relative_path"
msgstr "относительный_путь"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory. Example:
#. file:///some_dir
#: ../duplicity/commandline.py:838
msgid "some_dir"
msgstr "некоторый_каталог"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory where files will be
#. coming FROM. Example:
#. duplicity [full|incremental] [options] source_dir target_url
#: ../duplicity/commandline.py:844
msgid "source_dir"
msgstr "source_dir"

#. Used in usage help to represent a URL files will be coming
#. FROM. Example:
#. duplicity [restore] [options] source_url target_dir
#: ../duplicity/commandline.py:849
msgid "source_url"
msgstr "source_url"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory. where files will be
#. going TO. Example:
#. duplicity [restore] [options] source_url target_dir
#: ../duplicity/commandline.py:855
msgid "target_dir"
msgstr "target_dir"

#. Used in usage help to represent a URL files will be going TO.
#. Example:
#. duplicity [full|incremental] [options] source_dir target_url
#: ../duplicity/commandline.py:860
msgid "target_url"
msgstr "target_url"

#. Used in usage help to represent a user name (i.e. login).
#. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/commandline.py:870
msgid "user"
msgstr "пользователь"

#. Header in usage help
#: ../duplicity/commandline.py:895
msgid "Backends and their URL formats:"
msgstr "Заглушки и формат их URL:"

#. Header in usage help
#: ../duplicity/commandline.py:925
msgid "Commands:"
msgstr "Команды:"

#: ../duplicity/commandline.py:949
#, python-format
msgid ""
"Specified archive directory '%s' does not exist, or is not a directory"
msgstr "Заданная папка архива '%s' не существует или не является папкой"

#: ../duplicity/commandline.py:908
#, python-format
msgid ""
"Sign key should be an 8 character hex string, like 'AA0E73D2'.\n"
"Received '%s' instead."
msgstr ""
"Ключ подписи должен состоять из 8-и символов в шестнадцатеричном формате, "
"например 'AA0E73D2'.\n"
"Получено '%s'."

#: ../duplicity/commandline.py:1018
#, python-format
msgid ""
"Restore destination directory %s already exists.\n"
"Will not overwrite."
msgstr ""
"Каталог в который предназначено восстановление %s уже существует.\n"
"Замена не будет произведена."

#: ../duplicity/commandline.py:1023
#, python-format
msgid "Verify directory %s does not exist"
msgstr "Убедитесь, что папка %s не существует"

#: ../duplicity/commandline.py:1029
#, python-format
msgid "Backup source directory %s does not exist."
msgstr "Папка источника резервной копии %s не существует."

#: ../duplicity/commandline.py:1060
#, python-format
msgid "Command line warning: %s"
msgstr "Предупреждение командной строки: %s"

#: ../duplicity/commandline.py:1060
msgid ""
"Selection options --exclude/--include\n"
"currently work only when backing up,not restoring."
msgstr ""
"Параметры выбора --exclude/--include\n"
"в настоящее время работают только для резервного копирования, не для "
"восстановления."

#: ../duplicity/commandline.py:1110
#, python-format
msgid ""
"Bad URL '%s'.\n"
"Examples of URL strings are \"scp://user@host.net:1234/path\" and\n"
"\"file:///usr/local\".  See the man page for more information."
msgstr ""
"Недопустимая адресная ссылка URL '%s'.\n"
"Например: \"scp://user@host.net:1234/path\" и\n"
"\"file:///usr/local\".  Для получения дополнительных сведений обратитесь в "
"руководству (man)."

#: ../duplicity/commandline.py:1135
msgid "Main action: "
msgstr "Главное действие: "

#: ../duplicity/backend.py:102
#, python-format
msgid "Import of %s %s"
msgstr ""

#: ../duplicity/backend.py:211
#, python-format
msgid "Could not initialize backend: %s"
msgstr "Невозможно инициализиовать внутренний интерфейс: %s"

#: ../duplicity/backend.py:321
#, python-format
msgid "Attempt %s failed: %s: %s"
msgstr ""

#: ../duplicity/backend.py:371
#, python-format
msgid "Backtrace of previous error: %s"
msgstr ""

#: ../duplicity/backend.py:390
#, python-format
msgid "Attempt %s failed. %s: %s"
msgstr ""

#: ../duplicity/backend.py:386
#, python-format
msgid "Giving up after %s attempts. %s: %s"
msgstr ""

#: ../duplicity/backend.py:476
#, python-format
msgid "Reading results of '%s'"
msgstr "Чтение результата '%s'"

#: ../duplicity/backend.py:586
#, python-format
msgid "Running '%s' failed with code %d (attempt #%d)"
msgid_plural "Running '%s' failed with code %d (attempt #%d)"
msgstr[0] "Ошибка выполнения '%s' с кодом %d (попытка #%d)"
msgstr[1] "Ошибка выполнения '%s' с кодом %d (попытка #%d)"
msgstr[2] "Ошибка выполнения '%s' с кодом %d (попытка #%d)"

#: ../duplicity/backend.py:590
#, python-format
msgid ""
"Error is:\n"
"%s"
msgstr ""
"Ошибка:\n"
"%s"

#: ../duplicity/backend.py:592
#, python-format
msgid "Giving up trying to execute '%s' after %d attempt"
msgid_plural "Giving up trying to execute '%s' after %d attempts"
msgstr[0] "Прекращение попыток запуска '%s' после %d попытки"
msgstr[1] "Прекращение попыток запуска '%s' после %d попыток"
msgstr[2] "Прекращение попыток запуска '%s' после %d попыток"

#: ../duplicity/asyncscheduler.py:67
#, python-format
msgid "instantiating at concurrency %d"
msgstr "параллельное создание %d"

#: ../duplicity/asyncscheduler.py:94
msgid "inserting barrier"
msgstr "вставка барьера"

#: ../duplicity/asyncscheduler.py:143
msgid "running task synchronously (asynchronicity disabled)"
msgstr "выполнение задачи синхронно (асинхронность выключена)"

#: ../duplicity/asyncscheduler.py:149
msgid "scheduling task for asynchronous execution"
msgstr "Планирование задачи для асинхронного выполнения"

#: ../duplicity/asyncscheduler.py:178
msgid "task completed successfully"
msgstr "задача успешно завершена"

#: ../duplicity/asyncscheduler.py:189
msgid ""
"a previously scheduled task has failed; propagating the result immediately"
msgstr ""
"не удалось выполнить запланированную ранее задачу; выполняется мгновенное "
"распространение результата"

#: ../duplicity/asyncscheduler.py:212 ../duplicity/asyncscheduler.py:233
#, python-format
msgid "active workers = %d"
msgstr "активный обработчики = %d"

#: ../duplicity/asyncscheduler.py:253
#, python-format
msgid "task execution done (success: %s)"
msgstr "завершено выполнение задачи (успех: %s)"

#: ../duplicity/patchdir.py:80 ../duplicity/patchdir.py:85
#, python-format
msgid "Patching %s"
msgstr "Внесение изменений в %s"

#: ../duplicity/patchdir.py:532
#, python-format
msgid "Error '%s' patching %s"
msgstr ""

#: ../duplicity/patchdir.py:607
#, python-format
msgid "Writing %s of type %s"
msgstr "Запись %s типа %s"

#: ../duplicity/collections.py:161 ../duplicity/collections.py:172
#, python-format
msgid "BackupSet.delete: missing %s"
msgstr ""

#: ../duplicity/collections.py:197
msgid "Fatal Error: No manifests found for most recent backup"
msgstr ""
"Критическая ошибка: отсутствуют манифесты для недавней резервной копии"

#: ../duplicity/collections.py:206
msgid ""
"Fatal Error: Remote manifest does not match local one.  Either the remote "
"backup set or the local archive directory has been corrupted."
msgstr ""
"Критическая ошибка: Удалённый манифест не соответствует локальному. "
"Неисправен удалённый набор резервного копирования или папка локального "
"архива."

#: ../duplicity/collections.py:214
msgid "Fatal Error: Neither remote nor local manifest is readable."
msgstr ""
"Критическая ошибка: Нет доступа к чтению удалённого и локального манифеста."

#: ../duplicity/collections.py:327
msgid "Preferring Backupset over previous one!"
msgstr "Предпочтение набора резервного копирования вместо предыдущего!"

#: ../duplicity/collections.py:330
#, python-format
msgid "Ignoring incremental Backupset (start_time: %s; needed: %s)"
msgstr ""
"Пропуск инкрементального  набора резервного копирования (время_запуска: %s; "
"необходимо: %s)"

#: ../duplicity/collections.py:335
#, python-format
msgid "Added incremental Backupset (start_time: %s / end_time: %s)"
msgstr ""
"Добавлена пошаговая резервная копия (время_начала: %s / время_конца: %s)"

#: ../duplicity/collections.py:405
msgid "Chain start time: "
msgstr "Цепочка начата: "

#: ../duplicity/collections.py:406
msgid "Chain end time: "
msgstr "Цепочка завершена: "

#: ../duplicity/collections.py:407
#, python-format
msgid "Number of contained backup sets: %d"
msgstr "Количество содержащихся наборов резервного копирования: %d"

#: ../duplicity/collections.py:409
#, python-format
msgid "Total number of contained volumes: %d"
msgstr "Общее число содержащихся томов: %d"

#: ../duplicity/collections.py:411 ../duplicity/collections.py:1198
msgid "Type of backup set:"
msgstr "Тип резервной копии:"

#: ../duplicity/collections.py:411 ../duplicity/collections.py:1198
msgid "Time:"
msgstr "Время:"

#: ../duplicity/collections.py:411
msgid "Num volumes:"
msgstr "Число томов:"

#: ../duplicity/collections.py:415 ../duplicity/collections.py:1204
msgid "Full"
msgstr "Полная"

#: ../duplicity/collections.py:418 ../duplicity/collections.py:1206
msgid "Incremental"
msgstr "Пошаговая"

#: ../duplicity/collections.py:478
msgid "local"
msgstr "Локальная"

#: ../duplicity/collections.py:480
msgid "remote"
msgstr "удалённая"

#: ../duplicity/collections.py:635
msgid "Collection Status"
msgstr "Состояние сбора"

#: ../duplicity/collections.py:637
#, python-format
msgid "Connecting with backend: %s"
msgstr "Соединение с внутренним интерфейсом: %s"

#: ../duplicity/collections.py:639
#, python-format
msgid "Archive dir: %s"
msgstr "Папка архива: %s"

#: ../duplicity/collections.py:642
#, python-format
msgid "Found %d secondary backup chain."
msgid_plural "Found %d secondary backup chains."
msgstr[0] "Найдена %d вторичная цепочка резервных копий."
msgstr[1] "Найдены %d вторичные цепочки резервных копий."
msgstr[2] "Найдено %d вторичных цепочек резервных копий."

#: ../duplicity/collections.py:647
#, python-format
msgid "Secondary chain %d of %d:"
msgstr "Вторичная цепочка %d из %d:"

#: ../duplicity/collections.py:653
msgid "Found primary backup chain with matching signature chain:"
msgstr ""
"Найдена первичная цепочка резервного копирования с соответствующей цепочкой "
"подписи:"

#: ../duplicity/collections.py:657
msgid "No backup chains with active signatures found"
msgstr ""
"Отсутствуют цепочки резервного копирования с найденными, задействованными "
"подписями"

#: ../duplicity/collections.py:660
#, python-format
msgid "Also found %d backup set not part of any chain,"
msgid_plural "Also found %d backup sets not part of any chain,"
msgstr[0] ""
"Также найден %d набор резервного копирования, не являющегося частью ни одной "
"цепочки,"
msgstr[1] ""
"Также найдены %d набора резервного копирования, не являющихся частью ни "
"одной цепочки,"
msgstr[2] ""
"Также найдены %d наборов резервного копирования, не являющихся частью ни "
"одной цепочки,"

#: ../duplicity/collections.py:664
#, python-format
msgid "and %d incomplete backup set."
msgid_plural "and %d incomplete backup sets."
msgstr[0] "а также %d неполный набор резервного копирования."
msgstr[1] "а также %d неполных набора резервного копирования."
msgstr[2] "а также %d неполных наборов резервного копирования."

#. "cleanup" is a hard-coded command, so do not translate it
#: ../duplicity/collections.py:669
msgid ""
"These may be deleted by running duplicity with the \"cleanup\" command."
msgstr ""
"Это может быть удалено с помощью запуска duplicity с командой \"cleanup\"."

#: ../duplicity/collections.py:672
msgid "No orphaned or incomplete backup sets found."
msgstr ""
"Отсутствуют изолированные или неполные наборы резервного копирования."

#: ../duplicity/collections.py:688
#, python-format
msgid "%d file exists on backend"
msgid_plural "%d files exist on backend"
msgstr[0] "%d файл существует во внутреннем интерфейсе"
msgstr[1] "%d файлов существует во внутреннем интерфейсе"
msgstr[2] "%d файлов существует во внутреннем интерфейсе"

#: ../duplicity/collections.py:695
#, python-format
msgid "%d file exists in cache"
msgid_plural "%d files exist in cache"
msgstr[0] "%d файл существует во временных файлах"
msgstr[1] "%d файла существует во временных файлах"
msgstr[2] "%d файлов существует во временных файлах"

#: ../duplicity/collections.py:747
msgid ""
"Warning, discarding last backup set, because of missing signature file."
msgstr ""
"Внимание, отозван последний набор резервного копирования, в связи с "
"отсутствием файла подписи."

#: ../duplicity/collections.py:770
msgid "Warning, found the following local orphaned signature file:"
msgid_plural "Warning, found the following local orphaned signature files:"
msgstr[0] "Внимание, найден следующий локальный изолированный файл подписи:"
msgstr[1] ""
"Внимание, найдены следующие локальные изолированные файлы подписей:"
msgstr[2] ""
"Внимание, найдены следующие локальные изолированные файлы подписей:"

#: ../duplicity/collections.py:779
msgid "Warning, found the following remote orphaned signature file:"
msgid_plural "Warning, found the following remote orphaned signature files:"
msgstr[0] "Внимание, найден следующий удалённый изолированный файл подписи:"
msgstr[1] ""
"Внимание, найдены следующие удалённые изолированные файлы подписей:"
msgstr[2] ""
"Внимание, найдены следующие удалённые изолированные файлы подписей:"

#: ../duplicity/collections.py:788
msgid "Warning, found signatures but no corresponding backup files"
msgstr ""
"Внимание, найдены подписи, которые не соответствуют с файлами резервной копии"

#: ../duplicity/collections.py:792
msgid ""
"Warning, found incomplete backup sets, probably left from aborted session"
msgstr ""
"Внимание, обнаружены незавершённые наборы резервных копий, возможно они "
"остались от прерванного сеанса"

#: ../duplicity/collections.py:796
msgid "Warning, found the following orphaned backup file:"
msgid_plural "Warning, found the following orphaned backup files:"
msgstr[0] ""
"Внимание, найден следующий изолированный файл резервного копирования:"
msgstr[1] ""
"Внимание, найдены следующие изолированные файлы резервного копирования:"
msgstr[2] ""
"Внимание, найдены следующие изолированные файлы резервного копирования:"

#: ../duplicity/collections.py:813
#, python-format
msgid "Extracting backup chains from list of files: %s"
msgstr "Извлечение цепочек резервного копирования из списка файлов: %s"

#: ../duplicity/collections.py:824
#, python-format
msgid "File %s is part of known set"
msgstr "Файл %s является частью известного набора"

#: ../duplicity/collections.py:827
#, python-format
msgid "File %s is not part of a known set; creating new set"
msgstr "Файл %s не является частью известного набора; создаётся новый набор"

#: ../duplicity/collections.py:832
#, python-format
msgid "Ignoring file (rejected by backup set) '%s'"
msgstr "Пропуск файла (отклонено набором резервного копирования) '%s'"

#: ../duplicity/collections.py:848
#, python-format
msgid "Found backup chain %s"
msgstr "Найдена резервная копия %s"

#: ../duplicity/collections.py:853
#, python-format
msgid "Added set %s to pre-existing chain %s"
msgstr "Добавление набора %s для ранее существующей цепочки %s"

#: ../duplicity/collections.py:857
#, python-format
msgid "Found orphaned set %s"
msgstr "Обнаружен изолированный набор %s"

#: ../duplicity/collections.py:1008
#, python-format
msgid ""
"No signature chain for the requested time.  Using oldest available chain, "
"starting at time %s."
msgstr ""
"Отсутствует цепочка подписи для затребованного времени.  Использование "
"устаревшей доступной цепочки, время запуска %s."

#: ../duplicity/robust.py:61
#, python-format
msgid "Error listing directory %s"
msgstr "Ошибка перечня каталога %s"

#: ../duplicity/diffdir.py:108 ../duplicity/diffdir.py:398
#, python-format
msgid "Error %s getting delta for %s"
msgstr "Ошибка %s при получении различий для %s"

#: ../duplicity/diffdir.py:122
#, python-format
msgid "Getting delta of %s and %s"
msgstr "Приём дельты %s и %s"

#: ../duplicity/diffdir.py:167
#, python-format
msgid "A %s"
msgstr "Д %s"

#: ../duplicity/diffdir.py:174
#, python-format
msgid "M %s"
msgstr "И %s"

#: ../duplicity/diffdir.py:196
#, python-format
msgid "Comparing %s and %s"
msgstr "Сравнение %s и %s"

#: ../duplicity/diffdir.py:204
#, python-format
msgid "D %s"
msgstr "У %s"

#: ../duplicity/lazy.py:334
#, python-format
msgid "Warning: oldindex %s >= newindex %s"
msgstr "Внимание: старый индекс %s >= нового индекса %s"

#: ../duplicity/lazy.py:409
#, python-format
msgid "Error '%s' processing %s"
msgstr "Ошибка '%s' при обработке %s"

#: ../duplicity/lazy.py:419
#, python-format
msgid "Skipping %s because of previous error"
msgstr "Пропуск %s из-за предыдущей ошибки"

#: ../duplicity/backends/sshbackend.py:25
#, python-format
msgid ""
"Warning: Option %s is supported by ssh pexpect backend only and will be "
"ignored."
msgstr ""

#: ../duplicity/backends/sshbackend.py:33
#, python-format
msgid ""
"Warning: Selected ssh backend '%s' is neither 'paramiko nor 'pexpect'. Will "
"use default paramiko instead."
msgstr ""

#: ../duplicity/backends/giobackend.py:108
#, python-format
msgid "Connection failed, please check your password: %s"
msgstr "Не удалось подключиться, проверьте пароль: %s"

#: ../duplicity/backend.py:525
#, python-format
msgid "Writing %s"
msgstr "Записывается %s"

#: ../duplicity/manifest.py:91
#, python-format
msgid ""
"Fatal Error: Backup source host has changed.\n"
"Current hostname: %s\n"
"Previous hostname: %s"
msgstr ""
"Критическая ошибка: Узел источника резервной копии изменён.\n"
"Текущий узел: %s\n"
"Предыдущий узел: %s"

#: ../duplicity/manifest.py:98
#, python-format
msgid ""
"Fatal Error: Backup source directory has changed.\n"
"Current directory: %s\n"
"Previous directory: %s"
msgstr ""
"Критическая ошибка: Исходный каталог резервной копии был изменён.\n"
"Текущий каталог: %s\n"
"Предыдущий каталон: %s"

#: ../duplicity/manifest.py:107
msgid ""
"Aborting because you may have accidentally tried to backup two different "
"data sets to the same remote location, or using the same archive directory.  "
"If this is not a mistake, use the --allow-source-mismatch switch to avoid "
"seeing this message"
msgstr ""
"Прекращение в связи с тем, что вы можете случайно попытаться создать "
"резервные копии двух отличающихся наборов данных, по одному адресу, в одно и "
"тоже удалённое расположение.  Если это ошибка, используйте параметр --allow-"
"source-mismatch для скрытия этого сообщения"

#: ../duplicity/manifest.py:244
msgid "Manifests not equal because different volume numbers"
msgstr "Манифесты не одинаковы в связи с отличающимися номерами томов"

#: ../duplicity/manifest.py:249
msgid "Manifests not equal because volume lists differ"
msgstr "Манифесты не одинаковы в связи с отличающимися списками томов"

#: ../duplicity/manifest.py:254
msgid "Manifests not equal because hosts or directories differ"
msgstr "Манифесты не одинаковы в связи с отличающимися узлами или каталогами"

#: ../duplicity/manifest.py:401
msgid "Warning, found extra Volume identifier"
msgstr "Внимание, найден дополнительный идентификатор тома"

#: ../duplicity/manifest.py:427
msgid "Other is not VolumeInfo"
msgstr "Другой не является VolumeInfo"

#: ../duplicity/manifest.py:430
msgid "Volume numbers don't match"
msgstr "Номера томов не соответствуют"

#: ../duplicity/manifest.py:433
msgid "start_indicies don't match"
msgstr "start_indicies не соответствует"

#: ../duplicity/manifest.py:436
msgid "end_index don't match"
msgstr "end_index не соответствует"

#: ../duplicity/manifest.py:443
msgid "Hashes don't match"
msgstr "Хэши не совпадают"

#: ../duplicity/path.py:109
#, python-format
msgid "Warning: %s invalid devnums (0x%X), treating as (0, 0)."
msgstr ""

#: ../duplicity/path.py:236 ../duplicity/path.py:295
#, python-format
msgid "Warning: %s has negative mtime, treating as 0."
msgstr "Внимание: у %s отрицательное mtime, принимается равным 0."

#: ../duplicity/path.py:359
msgid "Difference found:"
msgstr "Найдено отличие:"

#: ../duplicity/path.py:368
#, python-format
msgid "New file %s"
msgstr "Новый файл %s"

#: ../duplicity/path.py:371
#, python-format
msgid "File %s is missing"
msgstr "Файл %s отсутствует"

#: ../duplicity/path.py:374
#, python-format
msgid "File %%s has type %s, expected %s"
msgstr "Файл %%s относится к типу %s, а ожидался %s"

#: ../duplicity/path.py:380 ../duplicity/path.py:406
#, python-format
msgid "File %%s has permissions %s, expected %s"
msgstr "Файл %%s имеет разрешения %s, а ожидалось %s"

#: ../duplicity/path.py:385
#, python-format
msgid "File %%s has mtime %s, expected %s"
msgstr "Файл %%s имеет mtime %s, а ожидалось %s"

#: ../duplicity/path.py:393
#, python-format
msgid "Data for file %s is different"
msgstr "Данные для файла %s отличаются"

#: ../duplicity/path.py:401
#, python-format
msgid "Symlink %%s points to %s, expected %s"
msgstr "Символическая ссылка %%s указывает на %s, а ожидалось %s"

#: ../duplicity/path.py:410
#, python-format
msgid "Device file %%s has numbers %s, expected %s"
msgstr "Файл устройства %%s содержит номера %s, а ожидались %s"

#: ../duplicity/path.py:572
#, python-format
msgid "Making directory %s"
msgstr "Создаётся каталог %s"

#: ../duplicity/path.py:582
#, python-format
msgid "Deleting %s"
msgstr "Удаляется %s"

#: ../duplicity/path.py:591
#, python-format
msgid "Touching %s"
msgstr ""

#: ../duplicity/path.py:598
#, python-format
msgid "Deleting tree %s"
msgstr "Удаление дерева %s"

#: ../duplicity/gpginterface.py:237
msgid "Threading not available -- zombie processes may appear"
msgstr ""

#: ../duplicity/gpginterface.py:701
#, python-format
msgid "GPG process %d terminated before wait()"
msgstr ""

#: ../duplicity/dup_time.py:52
#, python-format
msgid ""
"Bad interval string \"%s\"\n"
"\n"
"Intervals are specified like 2Y (2 years) or 2h30m (2.5 hours).  The\n"
"allowed special characters are s, m, h, D, W, M, and Y.  See the man\n"
"page for more information."
msgstr ""
"Строка с недопустимым интервалом \"%s\"\n"
"\n"
"Интервалы задаются на подобии 2Y (2 года) или 2h30m (2.5 часа).\n"
"Допустимые особые символы s (с.), m (м.), h (ч.), D(Д.), W (Н.), M (М.) и Y "
"(Г.).  \n"
"Обратитесь к руководству пользователя (man) для получения дополнительных "
"сведений."

#: ../duplicity/dup_time.py:58
#, python-format
msgid ""
"Bad time string \"%s\"\n"
"\n"
"The acceptible time strings are intervals (like \"3D64s\"), w3-datetime\n"
"strings, like \"2002-04-26T04:22:01-07:00\" (strings like\n"
"\"2002-04-26T04:22:01\" are also acceptable - duplicity will use the\n"
"current time zone), or ordinary dates like 2/4/1997 or 2001-04-23\n"
"(various combinations are acceptable, but the month always precedes\n"
"the day)."
msgstr ""
"Строка с недопустимым временем \"%s\"\n"
"\n"
"Допускается задание интервалов времени (например \"3D64s\"), строки \n"
"даты/времени формата w3, например \"2002-04-26T04:22:01-07:00\" (строки "
"типа\n"
"\"2002-04-26T04:22:01\" также допустимы - duplicity будет использовать\n"
"текущую временную зону) или обычные даты, например 2/4/1997 или 2001-04-23\n"
"(допустимы различные комбинации, но месяц всегда предшествует дню)."

#: ../duplicity/tempdir.py:132
#, python-format
msgid "Using temporary directory %s"
msgstr "Используется временный каталог %s"

#: ../duplicity/tempdir.py:176
#, python-format
msgid "Registering (mktemp) temporary file %s"
msgstr "Регистрация (mktemp) временного файла %s"

#: ../duplicity/tempdir.py:198
#, python-format
msgid "Registering (mkstemp) temporary file %s"
msgstr "Регистрация (mkstemp) временного файла %s"

#: ../duplicity/tempdir.py:230
#, python-format
msgid "Forgetting temporary file %s"
msgstr "Удаление временного файла %s"

#: ../duplicity/tempdir.py:233
#, python-format
msgid "Attempt to forget unknown tempfile %s - this is probably a bug."
msgstr ""
"Попытка удалить неизвестный временный файл %s - похоже, что это ошибка в "
"программе."

#: ../duplicity/tempdir.py:252
#, python-format
msgid "Removing still remembered temporary file %s"
msgstr "Удаление всё ещё нужного временного файла %s"

#: ../duplicity/tempdir.py:255
#, python-format
msgid "Cleanup of temporary file %s failed"
msgstr "Не удалось очистить временный файл %s"

#: ../duplicity/tempdir.py:260
#, python-format
msgid "Cleanup of temporary directory %s failed - this is probably a bug."
msgstr ""
"Не удалось очистить временный каталог %s - похоже, что это ошибка в "
"программе."

#: ../duplicity/util.py:99
#, python-format
msgid "IGNORED_ERROR: Warning: ignoring error as requested: %s: %s"
msgstr ""
"IGNORED_ERROR: Внимание: пропуск ошибки, как было обозначено в запросе: %s: "
"%s"

#: ../duplicity/util.py:165
#, python-format
msgid "Releasing lockfile %s"
msgstr ""

#~ msgid "Unable to load gio module"
#~ msgstr "Не удалось загрузить модуль gio"

#, python-format
#~ msgid ""
#~ "One only volume required.\n"
#~ "Renaming %s to %s"
#~ msgstr ""
#~ "Требуется только один том.\n"
#~ "Переименовывается %s в %s"

#, python-format
#~ msgid "Starting to write %s"
#~ msgstr "Начинается запись %s"

#~ msgid "Found old backup set at the following time:"
#~ msgid_plural "Found old backup sets at the following times:"
#~ msgstr[0] "Найдена резервная копия за дату:"
#~ msgstr[1] "Найдены резервные копии за дату:"
#~ msgstr[2] "Найдены резервные копии за дату:"

#~ msgid "Deleting backup set at time:"
#~ msgid_plural "Deleting backup sets at times:"
#~ msgstr[0] "Удаляется резервная копия за дату:"
#~ msgstr[1] "Удаляются наборы резервных копий за времена:"
#~ msgstr[2] "Удаляются резервные копии за даты:"
